third_party/ibis/ibis_addon/operations.py

```diff
@@ -507,7 +514,15 @@ def sa_cast_snowflake(t, op):
 
 
 def _sa_string_join(t, op):
-    return sa.func.concat(*map(t.translate, op.arg))
+    if (
+        len(op.arg) == 1
+    ):  # SQL Server concat errs on one argument (issue #1202), so casting to string - i.e. identity op
```

Comment:
Also a bit confused with this comment, not sure if it needs to be removed or rephrased

Reply:
Updated